src/styles/main.scss

```diff
@@ -1,14 +1,5 @@
-html {
-  font-family: Roboto, sans-serif;
-}
-
-body {
-  margin: 0;
-}
-
-* {
-  box-sizing: border-box;
-}
-
-@import "_header.scss";
-@import "_catalog.scss";
+@import "./blocks/pages.scss";
+@import "./utils/variables";
+@import "./utils/mixin";
+@import "./blocks/header";
+@import "./blocks/catalog";
```

Comment:
it would be better to use one main.scss
also main.scss is better to located outside the block folder